use_cases/eluc/predictors/sklearn/sklearn_predictor.py

```diff
@@ -0,0 +1,93 @@
+import json
+
+from pathlib import Path
+from abc import ABC
+
+import joblib
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+from sklearn.ensemble import RandomForestRegressor
+
+from predictors.predictor import Predictor
+
+class SKLearnPredictor(Predictor, ABC):
```

Comment:
The SKLearnPredictor pretty much handles everything since the SKLearn models all inherit from something behind the scenes that lets us do this.